connector/src/rest/connector/libs/nxos/implementation.py

```diff
@@ -103,17 +103,33 @@ def connect(self, timeout=30):
             >>> device = testbed.devices['asr22']
             >>> device.connect(alias='rest', via='rest')
         '''
+        protocol = 'https'
```

Comment:
why not do 

def connect(..., protocol='https'):

this way if people want to modify it at connection they can

Reply:
Updated as per comment